.github/workflows/stale.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/stale@v9
         with:
           stale-issue-message: 'This issue has been automatically closed because it has been inactive for 30 days. Please reopen if you think this issue is still relevant.'
-          days-before-issue-stale: 30
+          days-before-issue-stale: 90
           days-before-issue-close: 0
           stale-pr-message: 'The pull request is marked as stale because it has been inactive for 30 days. If there is no new activity it will be automatically closed in 5 days.'
```

Comment:
shouldn't this be 5, as in the message?

Reply:
I think you noticed this, but just to be clear. I thought closing an issue doesn't really need a warning as it can be reopened easily anyway. Don't mind either way if there is a good argument.

Reply:
ok, took your suggestion